### PR TITLE
jormungandr: handle starting/ending to a stop area in scenario experimental

### PR DESCRIPTION
**Pull request by kadhikari**
replace durations by 0 for stop_points of a stop_area

### CODE REVIEW

File: source/jormungandr/jormungandr/scenarios/experimental.py [hunk 1]
```diff
@@ -110,6 +110,17 @@ def create_parameters(request):
                              forbidden_uris=request['forbidden_uris[]'])
 
 
+def update_crowfly_duration(instance, fallback_list, mode, stop_area_uri):
+    if 'stop_area' not in stop_area_uri:
+        return
+    stop_points = instance.georef.get_stop_points_for_stop_area(stop_area_uri)
+    if not stop_points:
+        return
```
Comment (xlqian) [inline source/jormungandr/jormungandr/scenarios/experimental.py]: L117 and L118 are not necessary :)

File: source/jormungandr/jormungandr/georef.py [hunk 2]
```diff
@@ -126,3 +126,20 @@ def get_streetnetwork_routing_matrix(self, origins, destinations, streetnetwork_
...
+        req.ptref.count = 100
+        req.ptref.start_page = 0
+        req.ptref.depth = 1
+        req.ptref.filter = 'stop_area.uri = {uri}'.format(uri=uri)
+
+        result = self.instance.send_and_receive(req)
+        if len(result.stop_points) == 0:
+            logging.getLogger(__name__).info('PtRef, Unable to find stop_point with filter {}'.
+                                             format(req.ptref.filter))
+            return None
```
Comment (xlqian) [inline source/jormungandr/jormungandr/georef.py]: you can return [], and remove L117 and L118 in experimental.py

Comment (xlqian) [inline source/jormungandr/jormungandr/georef.py]: you can even remove L143 too

**Comment by kadhikari** [comment]
@xlqian  i made modifications on your comments

**Comment by xlqian** [comment]
Ok for me

**Comment by TeXitoi** [comment]
OK for me.

**Pull request merged**